### ISSUE TITLE
"cannot open input file 'libsndfile-1.lib'" on Windows

### ISSUE THREAD

**Issue by BenBergman** [roymacdonald/ofxSoundObjects#26]
I am trying to port a project of mine from macOS to Windows. Everything runs great on macOS but on Windows I am getting `cannot open input file 'libsndfile-1.lib'`. I have imported the .props file which has solved some other errors but I'm still stuck here. I see that libsndfile-1.lib exists in the cloned repo of ofxSoundObjects and the props file points to it, so I'm not really sure what to do to fix this.

**Comment by roymacdonald** [comment]
Hi, 
when do you get that error, at compilation time or run time. can you post what's being output by VS please?

**Comment by roymacdonald** [comment]
sorry, I think I made a mistake. I guess that the examples work but if you place your project in the apps folder it will not work. I will fix it, but on the meanwhile, you can manually change the `Additional Library Directories` in the project properties so it points to the folder where the `libsndfile-1.lib` is

**Comment by roymacdonald** [comment]
@BenBergman I've updated the `ofxSoundObjects.props` file. I've tested it with both the examples and a newly created project in the apps folder and in both cases it compiles and runs properly. Can you please test it and let me know if it fixes your issues?
cheers

**Comment by BenBergman** [comment]
Thanks for the pointer. The .props path being relative to the current project directory is the key I needed. I updated that and now my program compiles but it isn't playing audio. I tried running the `audioOutput` example (and several others) and I'm not getting any audio from there either. Audio files are clearly loading though - the audio player one shows a waveform and my own app is able to print out the metadata for the file. Any thoughts on that?
